.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,32 @@
+name: phpcs
+
+on:
```

Comment:
Are you able to match this to the `on` used in the `lint.yml` file?

If possible, I think it would be best to keep this in the existing file so all the linting is included in a single location.